cmd/check.go

```diff
@@ -27,7 +27,7 @@ import (
 
 var checker *ctl.CheckCommand
 
-func newCheckCommand(stdin io.Reader, stdout, stderr io.Writer) *cobra.Command {
+func newCheckCommand(_ io.Reader, _, _ io.Writer) *cobra.Command {
```

Comment:
oh dear... I think these are supposed to be passing stdin/out/err into the command below.